Makefile

```diff
@@ -214,3 +217,19 @@ qemu: $(qemu) $(bbl) $(rootfs)
 	$(qemu) -nographic -machine virt -kernel $(bbl) \
 		-drive file=$(rootfs),format=raw,id=hd0 -device virtio-blk-device,drive=hd0 \
 		-netdev user,id=net0 -device virtio-net-device,netdev=net0
+
+# Relevant partition type codes
+BBL   = 2E54B353-1271-4842-806F-E436D6AF6985
+LINUX = 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+FSBL  = 5B193300-FC78-40CD-8002-E86C45580B47
+
+.PHONY: format-boot-loader
+format-boot-loader: $(bin)
+	@test -b $(DISK) || echo "$DISK: is not a block device"
+	sgdisk --clear                                                               \
+		--new=1:2048:67583  --change-name=1:bootloader --typecode=1:$(BBL)   \
+		--new=2:264192:     --change-name=2:root       --typecode=2:$(LINUX) \
+		$(DISK)
+	@sleep 1
+	dd if=$< of=$(DISK)*1 bs=4096
```

Comment:
Why are there `*`s here?

Reply:
Because on linux it's /dev/sdc1, but on mac OS it's /dev/disk3s1